Documentation/kubernetes.md

```diff
@@ -119,6 +119,8 @@ To clean up all the artifacts created by the demo, run the following:
 kubectl delete -f wordpress.yaml
 kubectl delete -f mysql.yaml
 kubectl delete deployment rook-operator
```

Comment:
Does the order of deletion matter? Ie, should the rook-operator be deleted last (since it was the first one to be created) and the rookcluster second last?